ras/ras_extended.py

```diff
@@ -150,7 +150,7 @@ def test1_bootlist(self):
             cmd = "bootlist %s" % list_item
             self.run_cmd(cmd)
         interface = self.run_cmd_out(
-            "ifconfig | head -1 | cut -d':' -f1")
+            "ifconfig | head -1 | egrep '^(net|eth)' | cut -d':' -f1")
```

Comment:
Two questions here.
1. What if the interface is not named `net` or `eth`?
2. I find this script not handled for Suse because `ifconfig` is not available. Request it to be changed accordingly.